firmware/ventilator-controller-stm32/Core/Inc/Pufferfish/Driver/BreathingCircuit/AlarmMuteService.h

```diff
@@ -9,16 +9,26 @@
 
 #pragma once
 
-#include "Pufferfish/Application/LogEvents.h"
-#include "Pufferfish/Util/OrderedMap.h"
+#include "Pufferfish/Application/States.h"
 
 namespace Pufferfish::Driver::BreathingCircuit {
 
-static const float countdown_time = 120;
+static const uint64_t countdown_time = 120;  // s
```

Comment:
It looks like this constant is no longer used anywhere, so we can remove it.

Reply:
i'm using it for state_initializer https://github.com/pez-globo/pufferfish-software/pull/370/files#diff-ad9e425821d521acd9105056463dc5301d1d059e25c620755f2b15b5fa02568aR39 here, should i replace it with `mute_max_duration`, which is in `ms`

Reply:
Ah, I see. Yes, let's replace it with `mute_max_duration` - perhaps you can make that a `static const`. If our `AlarmMute` protobuf message definition uses units of seconds and that's making the firmware code more complicated, maybe we should change the `AlarmMute` protobuf message to use units of milliseconds, to be consistent with the units of time everywhere else in `mcu_pb.proto`? Let's discuss in today's meeting.